### ISSUE TITLE
Captive portal content with choice. Artifacts are left when going back to 1st choice =

### ISSUE THREAD

**Issue by jrouzierinverse** [inverse-inc/packetfence#2838]

**Comment by jrouzierinverse** [comment]
@julsemaan What do you think about that one?

![image](https://user-images.githubusercontent.com/2727473/34013319-f88b4b72-e0e4-11e7-8502-aa22448c9449.png)


![image](https://user-images.githubusercontent.com/2727473/34013317-f4209d08-e0e4-11e7-97c0-9a2ac83d26a4.png)

**Comment by julsemaan** [comment]
Hard to say exactly what it is, probably @cgx can tell you what is happening more than me

To me it doesn't look weird since a step forward was taken, creating a new chain, and then a step back was taken in that new chain that was created so it makes sense to keep the display of the chain in case you want to take a step forward in that chain instead of changing your mind.

Since you can't predict what the chain length will be before picking the choice, I think that's why its not displayed initially.

@cgx can confirm all of that or say that I've been writing non-sense for 5 minutes

**Comment by cgx** [comment]
That is correct. For the steps to be reset, you must click the little arrow at the top right corner of the portal.

**Comment by jrouzierinverse** [comment]
I find it confusing and unintuitive when you click on start over you still see your previous choice.
